src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -1692,12 +1711,12 @@ subroutine reflect(En, NAngle, CS, G, LB)
         angle_wall0 = angle_wall - 1
         ! compute relative angle from wall and use cyclic properties
         ! to ensure it is bounded by 0 -> Nangle-1
-        angle_to_wall = mod(a0 - angle_wall0 + Nangle, Nangle)
+        angle_to_wall = mod((a0 - angle_wall0) + Nangle, Nangle)
 
         if (ridge(i,j)) then
           ! if ray is not incident but in ridge cell, use complementary angle
-          if ((Nangle_d2 < angle_to_wall) .and. (angle_to_wall < Nangle)) then
-            angle_wall0 = mod(angle_wall0 + Nangle_d2 + Nangle, Nangle)
+          if ((Nangle_d2 .lt. angle_to_wall) .and. (angle_to_wall .lt. Nangle)) then
```

Comment:
Within the MOM6 code, we have a preference for the shorter and more language-invariant inequality comparison syntax `(a < b)` that was a part of the Fortran90 standard (and also appears in C, Java, Python, Matlab, etc.) instead of the older syntax `(a .lt. b)` that was required in Fortran77.  This is still legal Fortran code, and we can accept it, but do not be surprised when this specific syntax change gets reverted in a future sweep for strange syntax!